src/main/java/org/jabref/gui/edit/automaticfiededitor/copyormovecontent/CopyOrMoveFieldContentTabView.java

```diff
@@ -80,7 +81,7 @@ public Field fromString(String name) {
         toFieldComboBox.setConverter(new StringConverter<>() {
```

Comment:
Maybe reuse the https://github.com/JabRef/jabref/blob/d3e0f86a6be977aa16204fd611023e2358a556a7/src/main/java/org/jabref/gui/customentrytypes/CustomEntryTypeDialogViewModel.java#L40-L50

Then you have directly a Field object

Reply:
It seems like a ComboBox filled with fields is a very common use case; perhaps we should put it in its own class